bids/events/base.py

```diff
@@ -250,24 +250,25 @@ def __init__(self, base_dir, default_duration=0, default_amplitude=1,
         self.extra_columns = extra_columns
         self.sampling_rate = sampling_rate
 
-    def read(self, files=None, reset=True, **kwargs):
+    def read(self, file_directory=None, reset=True, **kwargs):
         ''' Read in and process event files.
         Args:
-            files (None, list): Optional list of event files to read from. If
-                None (default), will use all event files found within the
-                current BIDS project. Note that if `files` is passed, existing
-                event files within the BIDS project are completely ignored.
+            file_directory (None, str): Optional path to folder containing
```

Comment:
I'm not sure about this approach, because it requires all of the event files to be in the same folder, which is not at all guaranteed (e.g., I can imagine a common use case for this functionality being passing in the result of a wild card search through a directory tree). I'm wondering if maybe we should just drop support for extra-BIDS event files entirely for now. It seems like more trouble than it's worth. (Down the road, the way to address this is probably to refactor the classes in `base.py` so that they inherit from some non-BIDS-specific class that can handle arbitrary event file formats.)